w2w/w2w.py

```diff
@@ -443,6 +441,13 @@ def wrf_remove_urban(
     lat = dst_data.XLAT_M.squeeze()
     lon = dst_data.XLONG_M.squeeze()
     orig_num_land_cat = dst_data.NUM_LAND_CAT
+    is_urban = dst_data.ISURBAN
+    is_water = dst_data.ISWATER
+    is_lake = (
+        dst_data.ISLAKE
+        if hasattr(dst_data, 'ISLAKE') and 0 < dst_data.ISLAKE < orig_num_land_cat
+        else None
+    )
```

Comment:
```suggestion
    if hasattr(dst_data, 'ISLAKE') and 0 < dst_data.ISLAKE < orig_num_land_cat:
        is_lake = True
    else:
        is_lake = False
```
I am not a huge fan of inline if/else - i'd write it like that, but I'm also fine with that.

What I'm wondering though is why else `None` and not `False`? I'd make `is_lake` a proper boolean and not nullable

Reply:
Thanks for the review, for me it is fine to change the if/else structure.

Perhaps @lpilz or @dargueso can check the `None` or `False` statement?

Reply:
Hi, thanks for the review. I think the problem is the variable naming and I'll change that. Deriving the name from WRF's `ISLAKE`, `is_lake` is not a boolean, rather it's an integer, which is why I `None`'d it when it isn't defined. I use the `None` to selectively add some assignments e.g. in L472. I'll probably change the naming to something like `lake_cat`.